### DISCUSSION TITLE
Updating an app using scoop - what happens to installed plugins and extensions (e.g. visual code, eclipse)

### DISCUSSION THREAD

**Discussion by finrod2002** [ScoopInstaller/Scoop#5604] [Help and Questions] [answered]
Hello,

I wanted to ask what happens to the plugins and extensions installed in an application for example vscode or eclipse when updating the application itself via scoop.

As far as I can see when updating via scoop the new version of an application (e.g. vscode or eclipse) is installed in a new folder resulting in loosing currently installed plugins and extensions. 

What is the best practice to handle this situations?

**Comment by HUMORCE** [comment, marked as answer]
Manifests handle these situations with un|install scripts, usually.

e.g. vscode https://github.com/ScoopInstaller/Extras/pull/10074
